pyworkflow/protocol/protocol.py

```diff
@@ -1842,18 +1842,26 @@ def validate(self):
             childErrors = self._validate()
             if childErrors:
                 errors += childErrors
-        except Exception as e:
+        except Exception:
             import urllib
-            exceptionStr = pwutils.formatExceptionInfo(e)
+            exceptionStr = pwutils.formatExceptionInfo()
             email = pw.Config.SCIPION_SUPPORT_EMAIL
+            # In Python 3.x, you need to import urllib.parse.quote:
+            #
+            # import urllib.parse
+            # urllib.parse.quote("châteu", safe='')
+            # 'ch%C3%A2teu'
+            #
+            # The urllib module has been split into parts and renamed in Python 3 to urllib.request, urllib.parse,
+            # and urllib.error.
             errors.append("Sorry, this is embarrassing: the validation is "
```

Comment:
I think we can change this message, although true is a bit apocalyptic. What about:
"Validation has failed, this usually happens because there are some input missing. See is error message gives you any hint: \n %s" % exceptionStr

?